fitbenchmarking/fitbenchmarking
# Issue #264: Improve Parsing

#### Description of Work

Fixes #264


#### Testing Instructions

1. Compare master to this branch. Should be identical.


Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
